.github/workflows/app-scan.yml

```diff
@@ -0,0 +1,26 @@
+name: AppTemplate Scan
```

Comment:
would it make sense to rather scan the template that is exposed through Studio as well ?

Reply:
Maybe its better to just scan the template in studio, since that is "the truth" of which one is actually being used?